src/graphql/resolvers/token-resolver.ts

```diff
@@ -7,8 +7,8 @@ import { GraphQLContext } from "../context-type";
 
 @ArgsType()
 class TokensQueryArgs {
-  @Field(() => String, { nullable: true })
-  tokenId?: string;
```

Comment:
This will introduce a breaking change, maybe a better strategy is to flag it as deprecated and keep it for a while.

If `tokenId` is used you can insert it into `tokenIds` if it's not already present and remove it from `where` object. See [BoxRepository](https://github.com/capt-nemo429/ergo-graphql/blob/master/src/context/box-repository.ts#L37) for more informations.

Reply:
Done, thanks for the reference!